src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/CohostDocumentCompletionEndpointTest.cs

```diff
@@ -474,8 +474,8 @@ The end.
             completionContext: new RoslynVSInternalCompletionContext()
             {
                 InvokeKind = RoslynVSInternalCompletionInvokeKind.Typing,
-                TriggerCharacter = "f",
-                TriggerKind = RoslynCompletionTriggerKind.TriggerCharacter
+                TriggerCharacter = null,
+                TriggerKind = RoslynCompletionTriggerKind.Invoked
             },
```

Comment:
Here we (probably me) were specifying bad input data. I debugged and confirmed VS actually sends the newly specified data in this change

![image](https://github.com/user-attachments/assets/b4abd502-1cb2-4cee-a4cc-4918cfc8d428)